package.json

```diff
@@ -33,6 +33,7 @@
     "lodash": "~2.4.1",
     "mkdirp": "^0.5.0",
     "moment": "~2.7.0",
+    "otr": "^0.2.15",
```

Comment:
Not an expert, but seems fine to me, what we are doing here is either :

- linking (when distributing shout on source form)
- distribution in larger project (if distributing in a bundled form, but there is no such distribution of shout at the moment as far as I know)

[that wikipedia page](https://en.wikipedia.org/wiki/Comparison_of_free_and_open-source_software_licenses#General_comparison) states that linking to MPL 2.0 code is permissive, all license breakdowns I read like [this one](http://www.gatsbymedia.net/development-articles/why-we-use-the-mpl) or [this one](https://julien.ponge.org/blog/mozilla-public-license-v2-a-good-middleground/) states that MPL do not contaminates the whole project with MPL copyleft, but only the *files* of the work covered by MPL (in our case, the OTR lib, and optional change we would operate on it). 

I had to complete my license reading by 3rd-party analysis as raw law-geek-style license text is too vague to me.

What I don't manage to understand is if we should put some legal mention somewhere, but that's not a blocking point. Maybe some license-guru advice would be valuable, but IMHO we already know that we will not have to go backward on this choice for a licensing reason.